routes/quotes.js

```diff
@@ -88,7 +88,7 @@ var quotes = [
     'What ever you do, do it well. Do it so well that when people see you do it they will want to come back and see you do it again and they will want to bring others and show them how well you do what you do',
```

Comment:
you shouldn't mix the data for your application with the business logic to implement the different endpoints, they belong in separate places